.github/workflows/push_to_canary.yaml

```
@@ -97,21 +97,29 @@ jobs:
-            --google_credentials=bazel-cache-key.json 
-          bazel build //cloud_aws:cloud_aws_lib_deploy.jar \
-            --remote_cache=https://storage.googleapis.com/zipline-bazel-cache \
-            --google_credentials=bazel-cache-key.json 
-          bazel build //service:service_assembly_deploy.jar \
-            --remote_cache=https://storage.googleapis.com/zipline-bazel-cache \
-            --google_credentials=bazel-cache-key.json
-          bazel build //flink:connectors_pubsub_deploy.jar \
-            --remote_cache=https://storage.googleapis.com/zipline-bazel-cache \
-            --google_credentials=bazel-cache-key.json
+          if [ "${{ github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork }}" = "true" ]; then
+            bazel build //flink:flink_assembly_deploy.jar 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)